packages/dolt/content/introduction/cheat-sheet.md

```diff
@@ -0,0 +1,101 @@
+| ---                                             | -----                                   | --------                                                      |
+| `CREATE DATABASE mydb`                          | `dolt init`                             | Creates a new Dolt database                                   |
+| `CALL DOLT_CLONE('post-no-preference/options')` | `dolt clone post-no-preference/options` | Clones the `post-no-preference/options` database from DoltHub |
+
+## Stage and snapshot
+
+| SQL server                            | Dolt CLI                     | Comments                                      |
+| ---                                   | -----                        | --------                                      |
+| `CALL DOLT_ADD('myTable')`            | `dolt add myTable`           | Adds a table to the staging area              |
+| `CALL DOLT_RESET()`                   | `dolt reset`                 | Removes staged tables, keeps working changes  |
+| `CALL DOLT_RESET('--hard')`           | `dolt reset --hard`          | Resets all staged and working changes to HEAD |
+| `CALL DOLT_COMMIT('-m', 'a commit')`  | `dolt commit -m 'a commit'`  | Commits staged tables as a new snapshot       |
+| `CALL DOLT_COMMIT('-Am', 'a commit')` | `dolt commit -Am 'a commit'` | Stages and commits all tables                 |
+
+## Branch and merge 
+
+| SQL server                                     | Dolt CLI                    | Comments                                    |
+| ---                                            | -----                       | --------                                    |
+| `SELECT * FROM dolt_branches`                  | `dolt branch`               | Lists all branches                          |
+| `CALL DOLT_BRANCH('myBranch')`                 | `dolt branch myBranch`      | Creates a new branch                        |
+| `CALL DOLT_CHECKOUT('myBranch')`               | `dolt checkout myBranch`    | Switches to another branch                  |
+| `CALL DOLT_CHECKOUT('-b', 'myBranch')`         | `dolt checkout -b myBranch` | Creates a new branch and switches to it     |
+| `CALL DOLT_MERGE('myBranch')`                  | `dolt merge mybranch`       | Merges a branch into the checked out branch |
+
+## Diffing
+
+| SQL server                                                   | Dolt CLI                            | Comments                                                  |
+| ---                                                          | -----                               | --------                                                  |
+| `SELECT * FROM dolt_diff('HEAD', 'WORKING', 'mytable')`      | `dolt diff mytable`                 | Shows the working diff for `mytable`                      |
+| `SELECT * FROM dolt_diff_stat('HEAD', 'WORKING', 'mytable')` | `dolt diff --summary mytable`       | Shows statistics for the diff of `mytable`                |
```

Comment:
dolt diff summary shows a summary of data and schema changes. Could be worth mentioning here too?
```suggestion
| `SELECT * FROM dolt_diff_stat('HEAD', 'WORKING', 'mytable')` | `dolt diff --stat mytable`       | Shows statistics for the diff of `mytable`                |
```